modules/server_processes_manager/php/server_processes_manager.class.inc

```diff
@@ -114,9 +111,17 @@ class Server_Processes_Manager extends \NDB_Menu_Filter
         $deps    = parent::getJSDependencies();
         return array_merge(
             $deps,
-            array($baseURL . "/server_processes_manager/js/columnFormatter.js")
+            [
+             $baseURL . "/server_processes_manager/js/columnFormatter.js",
+             $baseURL . "/server_processes_manager/js/server_processes_managerIndex.js",
+            ]
         );
     }
 
+    function toJSON()
+    {
+        json_encode($this->toArray());
```

Comment:
this should `return json_encode($this->toArray())`, not just `json_encode($this->toArray());`

Reply:
seems like a reasonably good chance that this is what's broken.

Reply:
(also, this toJSON function could just be removed entirely since it does the same thing as the parent class.)